draft-packages/button/KaizenDraft/Button/styles.scss

```
@@ -103,18 +102,20 @@ $caButton-verticalPaddingForm: calc(
 
       // show custom focus ring when :focus-visible
       &:global(.focus-visible)::after {
-        $focus-ring-offset: ($kz-var-border-focus-ring-border-width * 2) + 1px;
+        $focus-ring-offset: calc(
+          (#{$kz-var-border-focus-ring-border-width}#{" * 2"}) #{" + 1px"}
```

Sass 🤮 

According to sass-meister this should work the same... can we try it?

```suggestion
          (#{$kz-var-border-focus-ring-border-width} * 2) + 1px
```

This style of syntax is pretty gross and I'm worried it'll scare away contributors, so if there is a saner syntax that still compiles I'd like to find it.

(Same goes for a bunch of other changes in this specific commit)

---

@jasononeil not sure if you saw my message but i'll fix these up and remove the extra literals